src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -519,7 +519,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			for (let i = 0; i < codeCells.length; i++) {
 				let cellStatus = await this.runCell(codeCells[i]);
 				if (!cellStatus) {
-					return Promise.reject(new Error(localize('cellRunFailed', 'running cell id {0} failed', codeCells[i].id)));
+					return Promise.reject(new Error(localize('cellRunFailed', 'Run Cells failed - See error in output of the currently select cell for more information.')));
```

Comment:
selected

Reply:
Also, in the localize method, the actual string displayed should be in double quotes instead of single quotes